smbus.cpp

```diff
@@ -0,0 +1,514 @@
+/*
+    i2c-dev.h - i2c-bus driver, char device interface
```

Comment:
Was this file newly written or copied from somewhere? The copyright header doesn't seem to match